ros2pkg/ros2pkg/verb/create.py

```diff
@@ -0,0 +1,179 @@
+            help='The package name')
+        parser.add_argument(
+            '--destination-directory',
+            default=os.path.realpath(os.getcwd()),
+            help='Directory where to create the package')
+        parser.add_argument(
+            '--build-tool',
+            default='ament_cmake',
+            choices=['cmake', 'ament_cmake'],
+            help='Which build tool to use')
+        parser.add_argument(
+            '--dependencies',
+            nargs='+',
+            default=[],
+            help='list of dependencies')
+        parser.add_argument(
+            '--maintainer-email',
+            default=getpass.getuser() + '@' + platform.uname()[1] + '.local',
+            help='email address of the maintainer of this package'),
+        parser.add_argument(
+            '--maintainer-name',
+            default=getpass.getuser(),
+            help='name of the maintainer of this package'),
+        parser.add_argument(
+            '--create-cpp-exe',
+            nargs='?',
+            const=True,
+            required=False,
+            help='create an empty cpp executable')
+        parser.add_argument(
+            '--cpp-exe-name',
+            nargs='+',
+            help='name of the empty cpp executable')
+        parser.add_argument(
+            '--create-cpp-library',
+            nargs='?',
+            const=True,
+            required=False,
+            help='create an empty cpp library')
+        parser.add_argument(
+            '--cpp-exe-library',
+            nargs='+',
+            help='name of the empty cpp library')
+
+    def main(self, *, args):
+        package_name = args.package_name
+        destination_directory = args.destination_directory
+        build_tool = args.build_tool
+        dependencies = args.dependencies
+        maintainer_email = args.maintainer_email
+        maintainer_name = args.maintainer_name
+        create_cpp_exe = True if args.create_cpp_exe else False
+        cpp_exe_name = (
+            package_name + '_main.cpp' if not args.cpp_exe_name else args.cpp_exe_name)
+        create_cpp_library = True if args.create_cpp_library else False
+        cpp_library_name = (
+            package_name + '.cpp' if not args.cpp_exe_name else package_name + '.cpp')
+
+        print('going to create a new package')
+        print('package name:', package_name)
+        print('destination directory', destination_directory)
+        print('build tool:', build_tool)
+        print('maintainer_email:', maintainer_email)
+        print('maintainer_name:', maintainer_name)
+        print('dependencies:', dependencies)
+        print('create_cpp_exe:', create_cpp_exe)
+        print('cpp_exe_name:', cpp_exe_name)
+        print('create_cpp_library:', create_cpp_library)
+        print('cpp_library_name:', cpp_library_name)
+
+        package_directory = create_folder(package_name, destination_directory)
+        if not package_directory:
+            return
```

Comment:
There is no error code for this case. The caller can't determine that the script failed.